lib/mongify/translation/sync.rb

```diff
@@ -63,7 +63,8 @@ def sync_data
           max_updated_at, max_updated_at_id = Time.new(1970), nil
           rows.each do |row|
             row_hash = t.translate(row)
-            updated_at = Time.parse(row['updated_at'])
+            updated_at = row['updated_at']
+            updated_at = Time.parse(updated_at) if updated_at.instance_of?(String)
```

Comment:
Line is too long. [82/80]<br>Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.